sauce/features/accounts/showCategoryBalance/settings.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  name: 'ShowCategoryBalance',
+  type: 'checkbox',
+  default: false,
+  section: 'accounts',
```

Comment:
yeah, I think you're right. I'll add this as a todo to the PR as I might not get to it right away.